languages.toml

```diff
@@ -1781,3 +1781,25 @@ language-server = { command = "bass", args = ["--lsp"] }
 [[grammar]]
 name = "bass"
 source = { git = "https://github.com/vito/tree-sitter-bass", rev = "501133e260d768ed4e1fd7374912ed5c86d6fd90" }
+
+[[language]]
+name = "wat"
+scope = "source.webassembly"
+comment-token = ";;"
+file-types = ["wat"]
+roots = []
+
+[[grammar]]
+name = "wat"
+source = { git = "https://github.com/wasm-lsp/tree-sitter-wasm", rev = "2ca28a9f9d709847bf7a3de0942a84e912f59088", subpath = "wat" }
+
+[[language]]
+name = "wast"
+scope = "source.webassembly"
```

Comment:
The scopes are the same between these two. I think wat could be `source.webassembly.text` but I'm not sure what to use for wast

Reply:
I've tentatively set this to `source.wat`, which is the scope used by the main extensions in Sublime and VSCode. But not strong opinions here. If we were to follow your suggestion, we could set `source.webassembly.script`?

Reply:
We could use `source.webassembly.text` & `source.webassembly.script`. Since Sublime and VSCode use these though, I would prefer to also use `source.wat` & `source.wast`.